frontend/public/components/utils/headings.tsx

```diff
@@ -170,6 +172,7 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
             className={classNames({
               'co-m-pane__heading--logo': props.icon,
               'co-m-pane__heading--with-help-text': helpText,
+              'co-m-pane__heading--full-width': !showActions,
```

Comment:
```suggestion
```

Reply:
If I remove this style,then for list pages, icon will not align properly

<img width="904" alt="image" src="https://github.com/user-attachments/assets/0ef99bc7-8105-4301-8f4b-900229008ff3" />